test/scripts/e2e-cp.sh

```diff
@@ -124,7 +124,7 @@ fi
 
 # Only run Node IP/MAC address migration tests if they are explicitly requested
 IP_MIGRATION_TESTS="Node IP and MAC address migration"
-if [ "${WHAT}" != "${IP_MIGRATION_TESTS}" ]; then
+if [[ "${WHAT}" != "${IP_MIGRATION_TESTS}"* ]]; then
```

Comment:
Why do we need these changes?

Reply:
that allows you running tests with WHAT="full test spec". e.g. now if you call it with
WHAT="Network Segmentation a user defined primary network created using NetworkAttachmentDefinitions"
it won't get executed, because it searched for an exact "Network Segmentation" match.
I use it often locally to run specific tests and I have already done this for some tests https://github.com/ovn-org/ovn-kubernetes/blob/master/test/scripts/e2e-cp.sh#L136-L142

Reply:
hmm looks we have started miss-using $WHAT. Yours is the expected use case, it was not to focus on a set of well known tests.